### PR TITLE
Updating scripts/genproto.sh to refer to the correct folders on finish

### PR DESCRIPTION
**Pull request by nate-double-u**
scripts/genproto.sh now points the user to the `/content/en/docs/v3.5/dev-guide/` folder on completion (`next` folder has been removed in the website)

Contributes to https://github.com/etcd-io/etcd/issues/13080

### CODE REVIEW

**Comment by ptabor** [comment]
I'm sorry if missed some decisions about layout of the files in: https://github.com/etcd-io/website/tree/main/content/en/docs. 

I think that `release-3.5` branch should point on docs in v3.5 directory. For the `main` branch we should be targeting
`https://github.com/etcd-io/website/tree/main/content/en/docs/next` (that temporarily does not exists yet but eventually should contain payload for v3.6) such that all API changes provided for v3.6 will not lend in v3.5 docs.

**Comment by ptabor** [comment]
Thank you for the pointer. 

In the new model `main` should point on '3.6' directory, such that API changes specific to v3.6 will not pollute the v3.5 docs, right ?

**Comment by nate-double-u** [comment]
> I think that release-3.5 branch should point on docs in v3.5 directory. For the main branch we should be targeting
https://github.com/etcd-io/website/tree/main/content/en/docs/next (that temporarily does not exists yet but eventually should contain payload for v3.6) such that API changes developed for v3.6 will not lend in v3.5 docs.

I think the release-3.5 branch has the old/original style `scripts/genproto.sh`, from before we made [these changes](https://github.com/etcd-io/etcd/pull/13062). I can open a PR to bring release-3.5 in line with main (adding the info about v3.5's location), and then close this PR as unneeded if we [agree that `next` will remain how we manage upcoming versions](https://github.com/etcd-io/website/issues/252).

File: scripts/genproto.sh [hunk 1]
```diff
@@ -131,8 +131,8 @@ This API reference is autogenerated from the named \`.proto\` files." || exit 2
   log_success -e "\\nTo publish the API references, copy the following files"
   log_success "  - ${API_REFERENCE_FILE}"
   log_success "  - ${API_REFERENCE_CONCURRENCY_FILE}"
-  log_success "to the etcd-io/website repo under the /content/en/docs/next/dev-guide/ folder."
-  log_success "(https://github.com/etcd-io/website/tree/main/content/en/docs/next/dev-guide)"
+  log_success "to the etcd-io/website repo under the /content/en/docs/v3.5/dev-guide/ folder."
+  log_success "(https://github.com/etcd-io/website/tree/main/content/en/docs/v3.5/dev-guide)"
```
Comment (nate-double-u) [inline scripts/genproto.sh:135]: > In the new model `main` should point on '3.6' directory, such that API changes specific to v3.6 will not pollute the v3.5 docs, right ?

I think for now, we should leave these lines alone, but eventually point them to `v3.6`.

As of right now, neither `next` nor `v3.6` exist in the folder structure of the Docs.

**Comment by nate-double-u** [comment]
closing in favour of https://github.com/etcd-io/etcd/pull/13120

**Pull request closed without merging**